Update feature-release.md (#691)
mattermost/mattermost-handbook: operations/research-and-development/product/release-process/feature-release.md

    - Confirm the required URLs for the blog/release announcement were prepared
3. Logistics @hanna.park:
    - Mail out contributor and security researcher mugs
      - Space out the ordering of mugs over the next three weeks to prevent mistakes being made by the supplier (e.g., If there are 12 contributors to order mugs for, place an order every 2nd or 3rd day over the next three weeks)
    - Update [Team](http://www.mattermost.org/team/) page with new contributors
    - Generate an E20 5000 seat test licence and email to Lindy for release testing
4. QA:
    - Confirm up to date with testing PRs and resolved tickets
    - Confirm up to date with test updates and known issues in release test management and automated tests
    - Assign release testing areas to team members
    - After RC1 is cut: Update test server invite links in Release Testing instructions
    - After RC1 is cut: Run automated regression tests

>     - Post reminders in Release Announcements channel ([template](https://docs.google.com/document/d/1UvTsvwZXmEL9QPjxmjoIkR2AcwGtOjql8cYRfk2N8eA/edit#bookmark=id.pirw51cwsja5)) for Mobile App RC testing

Reviewer: ```suggestion
    - Post reminder in Release Announcements channel for any unfinished RC testing, at-mentioning tester(s)
```

       - Find QA or other teammates to help finish unfinished tests if needed
       - Verify all release tests are finished, bring any concerns to Triage/Release meeting

>        - Go through all tabs of testing spreadsheet and verify all comments and questions have been filed in Jira as needed

Reviewer: ```suggestion
       - Go through test cycles and verify all issues and comments have been filed or updated in Jira as needed
```
